src/cascadia/WindowsTerminal/pch.h

```diff
@@ -30,6 +30,8 @@ Module Name:
 #include <shellscalingapi.h>
 #include <windowsx.h>
 
+// Manually include til after we include Windows.Foundation to give it winrt superpowers
+#define BLOCK_TIL

```

Comment:
Can you file a follow-on task to perhaps make a second `LibraryIncludes.h` that includes WIL and TIL and have it at the bottom of every `precomp.h`?

I feel like you've done this to like 6 more projects as of this PR and I bet it will just keep coming.

We should probably put all super-powered headers at the bottom of every `precomp.h`.

Reply:
Ask and you shall receive: #5182